dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/CollectionDAO.java

```diff
@@ -44,6 +44,22 @@ public Collection findByNameAndOrg(String name, long organizationId) {
         return uniqueResult(query);
     }
 
+    public void deleteCollectionByOrgId(long organizationId) {
```

Comment:
This can delete many collections, can't it? I was confused at first where you invoke it because I was wondering why you were only deleting one collection. I would call it "deleteCollectionsByOrgId"

Reply:
it can delete multiple collections, renamed